pkg/storage/etcd3/watcher.go

```diff
@@ -168,6 +168,8 @@ func (wc *watchChan) startWatching() {
 		opts = append(opts, clientv3.WithPrefix())
 	}
 	wch := wc.watcher.client.Watch(wc.ctx, wc.key, opts...)
+	// On normal running, we should never stop the watching loop.
+	defer glog.Warningf("stop watching key (%s) from etcd.", wc.key)
```

Comment:
What does this mean, why is it a warning, and why is it always printed? clearly watches will be ended at some point?